src/Unosquare.Swan/Components/ObjectMap.cs

```diff
@@ -0,0 +1,125 @@
+﻿namespace Unosquare.Swan.Components
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Linq.Expressions;
+    using System.Reflection;
+
+    /// <summary>
+    /// Represents an object map
+    /// </summary>
+    /// <typeparam name="TSource">The type of the source.</typeparam>
+    /// <typeparam name="TDestination">The type of the destination.</typeparam>
+    /// <seealso cref="Unosquare.Swan.Components.IObjectMap" />
+    public class ObjectMap<TSource, TDestination> : IObjectMap
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ObjectMap{TSource, TDestination}" /> class.
+        /// </summary>
+        /// <param name="intersect">The intersect.</param>
+        public ObjectMap(IEnumerable<PropertyInfo> intersect)
+        {
+            SourceType = typeof(TSource);
+            DestinationType = typeof(TDestination);
+
+            Map = new Dictionary<PropertyInfo, List<PropertyInfo>>();
+
+            foreach (var property in intersect)
```

Comment:
https://msdn.microsoft.com/en-us/library/bb549277(v=vs.110).aspx

Reply:
?

Reply:
could use ToDictionary instead of foreach